README.md

```diff
@@ -106,15 +111,13 @@ For frequently asked questions and their answers, please visit our [FAQ page](ht
 
 ## 🤝 Contributing
 
-Contributions, issues and feature requests are welcome. After cloning and setting up project locally, you can just submit 
-a PR to this repo and it will be deployed once it's accepted.
+Contributions, issues and feature requests are welcome. After cloning and setting up a project locally, you can just submit a PR to this repo and it will be deployed once it's accepted.
```

Comment:
hmm, a project, or *the* project?

Reply:
Fair point, I just spotted an article missing. I guess it applies to the templates as well? But the may be in place here, yeah.